src/pivotal-ui/components/panels/panels.scss

```diff
@@ -130,175 +157,14 @@ Basic alternate panels are panels with a border and shadow. You can have a panel
 .panel-basic-alt {
   border: $panel-basic-alt-border;
   border-radius: $panel-basic-alt-border-radius;
+  overflow: hidden;
```

Comment:
oooh, interesting. this is a little bit dangerous because it will chop off other flyout menus, dropdowns, etc... And yet, would people building it expect it to be the same for all panels and not just panel-basic-alt? Hard questions.